src/components/PaginationV2/PaginationV2.js

```diff
@@ -325,8 +326,9 @@ export default class PaginationV2 extends Component {
             <Select
               id={`${prefix}-pagination-select-${inputId + 2}`}
               className={`${prefix}--select__page-number`}
-              hideLabel
+              labelText={`Page number, of ${totalPages} pages`}
```

Comment:
g11n - Can we extract out this template as a prop (probably a form of function)? Thanks!